src/templates/digitalMedia/digitalMedia.scss

```diff
@@ -0,0 +1,147 @@
+/* Custom styling for digital specimen page */
+
+.digitalMedia_body {
+    height: Calc(100% - 147px);
+}
+
+.digitalMedia_mediaBody {
+    height: 70%;
+}
+
+.digitalMedia_titleBlock {
+    height: 40px;
+    line-height: 40px;
+    font-size: 20px;
+}
+
+.digitalMedia_mediaBodyImages {
+    height: Calc(100% - 40px) !important;
+}
+
+.digitalMedia_mediaBodyTab {
+    height: 40px;
+    line-height: 40px;
+    background-color: var(--greyXLight);
+    cursor: pointer;
+}
+
+.digitalMedia_mediaBodyTab.active {
+    background-color: var(--main-color-dark);
+    color: white;
+    cursor: default;
+}
+
+.digitalMedia_mediaBodyContent {
+    display: none;
+}
+
+.digitalMedia_mediaBodyContent.active {
+    display: block;
+}
+
+.digitalMedia_mediaMetaContent {
+    height: Calc(100% - 40px);
+}
+
+
+/* Media List */
+
+.digitalMedia_mediaListItem {
+    height: 80px;
+}
+
+.digitalMedia_mediaListItem.hover {
+    cursor: pointer;
+}
+
+.digitalMedia_mediaListItemImage.hover {
+    background-color: white;
+    opacity: 0.8;
+}
+
+.digitalMedia_mediaListItemText {
+    display: none;
+}
+
+.digitalMedia_mediaListItemText.hover {
+    display: initial;
+}
+
+.digitalMedia_mediaListItemText.chosen {
+    display: initial;
+    color: white;
+}
+
+
+/* Media Attributes */
+
+.digitalMedia_attributeBlock {
+    font-size: 14px;
+    transition: 0.4s;
+    cursor: pointer;
+}
+
+.digitalMedia_attributeBlock:hover {
+    background-color: var(--greyLight);
+}
+
+
+/* Media Annotations */
+
+.digitalMedia_annotationsBlock {
+    font-size: 14px;
+}
+
+.digitalMedia_annotationsBlockHeader {
+    background-color: var(--greyLight);
+    font-size: 14px;
+}
+
+.digitalMedia_annotationsBlockRow {
+    transition: 0.4s;
+    cursor: pointer;
+}
+
+.digitalMedia_annotationsBlockRow:hover {
+    background-color: var(--main-color-light);
+}
+
+
+/* Media Modal */
+
+.digitalMedia_modal .modal-dialog {
+    height: 100%;
+    margin: 0 auto !important;
+}
+
+.digitalMedia_modal .modal-content {
+    background-color: rgba(255, 255, 255, 0);
+    height: 84.8%;
+    overflow: hidden;
+    padding-top: 11.2%;
+    border: none;
+}
+
+
+/* Media Modal Menu */
+
+.digitalMedia_modalMenuTitle {
+    height: 41px;
+    line-height: 41px;
+}
+
+.digitalMedia_modalCloseButton {
+    cursor: pointer;
+}
+
+
+/* Media Modal Image Annotate */
+
+.hAjSKD {
```

Comment:
Are these names correct, are they abbreviations or just random?

Reply:
Yeah, they are classes used by the image annotation library, found them by inspecting cause I needed an inner element to stretch to the correct width and height which I could not achieve by simply passing classes onto the element, will give add a more clearer description in the CSS file

Reply:
Ok maybe leave an additional comment in the code with this so future dev's will understand

Reply:
Ah I now see the change, ok should be enough